server/sonar-server/src/main/java/org/sonar/server/user/ws/HomepageTypesImpl.java

```diff
@@ -66,23 +65,29 @@ public HomepageTypesImpl(Configuration configuration, OrganizationFlags organiza
 
   @Override
   public Type getDefaultType() {
-    checkState(types != null, "Homepage types  have not been initialized yet");
-    return defaultType;
+    return isOnSonarCloud() ? MY_PROJECTS : PROJECTS;
   }
 
   @Override
   public void start() {
     try (DbSession dbSession = dbClient.openSession(false)) {
-      boolean isOnSonarCloud = configuration.getBoolean(ProcessProperties.Property.SONARCLOUD_ENABLED.getKey()).orElse(false);
-      boolean isOrganizationEnabled = organizationFlags.isEnabled(dbSession);
+      boolean isOnSonarCloud = isOnSonarCloud();
+      boolean isOrganizationEnabled = isOrganizationEnabled(dbSession);
```

Comment:
I thought you had issues with having the organization flag at start up?

Reply:
no more since we decided to ignore validation in framework definition